config-service/src/main/resources/db/migration/V002___init.sql

```diff
@@ -0,0 +1,21 @@
+create table users_table
+(
+    id         bigserial primary key not null,
```

Comment:
Change id type to uuid with autogeneration

```suggestion
    id uuid primary key default uuid_generate_v4()
```